TicTacToe/TicTacToe.cs

```diff
@@ -27,61 +28,175 @@ public static void Main()
 
         public static void GetInput()
         {
+            //Ternary Operator for taking turns
```

Comment:
You are swapping the playerTurn variable in too many places. While the logic seems to be correct, and you are swapping only when needed, the fact that it is beeing swapped so many times is a little odd.